frontend/src/modules/Traces/TraceGanttChart/TraceGanttChart.tsx

```diff
@@ -229,24 +229,26 @@ const TraceGanttChart = ({
 	};
 
 	const handleResetFocus = () => {
-		let rows = document.querySelectorAll("#collapsable table tbody tr");
+		const rows = document.querySelectorAll('#collapsable table tbody tr');
 		Array.from(rows).map((row) => {
-			row.classList.remove("hide");
+			row.classList.remove('hide');
 		});
 
 		resetZoom(true);
 	};
 
-	const handleScroll = (id) => {
-		let rows = document.querySelectorAll("#collapsable table tbody tr");
-		const table = document.querySelectorAll("#collapsable table");
-		Array.from(rows).map((row) => {
-			let attribKey = row.getAttribute("data-row-key");
-			if (id === attribKey) {
-				let scrollValue = row.offsetTop;
-				table[1].scrollTop = scrollValue;
-			}
-		});
+	const handleScroll = (id: string): void => {
+		if (!isEmpty(id)) {
+			const rows = document.querySelectorAll<HTMLElement>('#collapsable table tbody tr');
+			const table = document.querySelectorAll<HTMLElement>('#collapsable .ant-table-body');
+			Array.from(rows).map((row) => {
+				const attribKey = row.getAttribute('data-row-key');
```

Comment:
Cant, we get the row by `data-attributes` selector ? 

```
document.querySelectorAll(`[data-row-key=${id}]`)
```

Reply:
Both methods are correct. Can use any

Reply:
I maybe missing something here. But when we use select all elements with specific row key we can directly scroll to that element without looking over all rows. Another reason being we rely on antd generated class and ids, if in near future antd upgrade changes them (very low probabilty) it can break the functionality.

Reply:
Any suggestions you have over using antd classes and ids?

Reply:
How about using:

```
selectedRow[0].scrollIntoView()
```

This way we can remove table selector as well.